File: 10214.py
for i in range(int(input())):
    yonsei = 0; korea = 0
    for j in range(9):
        Y, K = map(int, input().split())
        yonsei += Y; korea += K

    if yonsei > korea: print("Yonsei")
    elif yonsei < korea: print("Korea")
    else: print("Draw")

File: 10214.cpp
#include <iostream>
using namespace std;

int main() {
	int t; cin >> t;

	for (int i = 0; i < t; i++) {
		int yonsei = 0, korea = 0;
		for (int i = 0; i < 9; i++) {
			int y, k; cin >> y >> k;
			yonsei += y; korea += k;
		}

		if (yonsei > korea) cout << "Yonsei\n";
		else if (yonsei < korea) cout << "Korea\n";
		else cout << "Draw\n";
	}
}
